aikau/src/test/resources/alfresco/services/DialogServiceTest.js

```diff
@@ -46,440 +46,504 @@ define(["intern!object",
-            .then(null, function() {
-               assert(false, "The Dialog did not appear");
-            });
-      },
-
-      "Test publication on dialog show": function() {
-         return browser.findByCssSelector(".alfresco-dialog-AlfDialog")
-         .end()
-         .getLastPublish("DISPLAYED_FD1", "Could not find topic published when displayed dialog");
-      },
-
-      "Test recreating dialog with no ID": function() {
-         return closeAllDialogs(browser)
-            .then(function() {
-               return browser.findById("CREATE_FORM_DIALOG_NO_ID")
-                  .click()
+            .findByCssSelector(".alfresco-dialog-AlfDialog")
+               .then(null, function() {
+                  assert(false, "The Dialog did not appear");
+               });
+         },
+
+         "Test publication on dialog show": function() {
+            return browser.findByCssSelector(".alfresco-dialog-AlfDialog")
+            .end()
+            .getLastPublish("DISPLAYED_FD1", "Could not find topic published when displayed dialog");
+         },
+
+         "Test recreating dialog with no ID": function() {
+            return closeAllDialogs(browser)
+               .then(function() {
+                  return browser.findById("CREATE_FORM_DIALOG_NO_ID")
+                     .click()
+                     .end()
+
+                  .findAllByCssSelector(".alfresco-dialog-AlfDialog")
+                     .then(function(elements) {
+                        assert.lengthOf(elements, 1, "The previous dialog without an ID was not destroyed");
+                     });
+               });
+         },
+
+         "Test creating dialog with an ID": function() {
+            return closeAllDialogs(browser)
+               .then(function() {
+                  return browser.findById("CREATE_FORM_DIALOG")
+                     .click()
+                     .end()
+
+                  .findAllByCssSelector(".alfresco-dialog-AlfDialog")
+                     .then(function(elements) {
+                        assert.lengthOf(elements, 2, "The previous dialog (without an ID) was destroyed");
+                     });
+               });
+         },
+
+         "Test recreating dialog with an ID": function() {
+            return closeAllDialogs(browser)
+               .then(function() {
+                  return browser.findById("CREATE_FORM_DIALOG")
+                     .clearLog()
+                     .click()
                   .end()
 
-               .findAllByCssSelector(".alfresco-dialog-AlfDialog")
-                  .then(function(elements) {
-                     assert.lengthOf(elements, 1, "The previous dialog without an ID was not destroyed");
-                  });
-            });
-      },
+                  .findAllByCssSelector(".alfresco-dialog-AlfDialog")
+                     .then(function(elements) {
+                        assert.lengthOf(elements, 2, "The previous dialog (without an ID) was destroyed");
```

Comment:
Your recent commit, I think? If you have a look at the differences with whitespace ignored, then this change disappears, so I think it's just do with re-indenting the file.